tests/tests/test-gas/test-gas-contract-creation.ts

```diff
@@ -12,6 +12,6 @@ describeDevMoonbeam("Estimate Gas - Contract creation", (context) => {
         from: GENESIS_ACCOUNT,
         data: contract.byteCode,
       })
-    ).to.equal(127607);
+    ).to.equal(150926);
```

Comment:
@tgmichel we should have followed your instinct and spend more time on it when you found this when unexpected :)